StandardConfig/production/ILDReconstruction.py

```diff
@@ -219,22 +223,28 @@ def create_reader(input_files):
 ecal_technology = CONSTANTS["EcalTechnology"]
 hcal_technology = CONSTANTS["HcalTechnology"]
 
-sequenceLoader.load("Tracking/TrackingDigi")
-sequenceLoader.load("Tracking/TrackingReco")
-sequenceLoader.load(f"CaloDigi/{ecal_technology}Digi")
-sequenceLoader.load(f"CaloDigi/{hcal_technology}Digi")
-sequenceLoader.load("CaloDigi/FcalDigi")
-sequenceLoader.load("CaloDigi/MuonDigi")
-
-if reco_args.perfectPFA:
-    sequenceLoader.load("ParticleFlow/PandoraPFAPerfect")
+if reco_args.detectorModel in FCCeeMDI_DETECTOR_MODELS:
```

Comment:
This check will fail if someone uses the `--compactFile` option to pass in a `DetectorModel` that is not in the installed k4geo prefix.

Reply:
That's right. Do you have any ideas on how to solve it properly?

Reply:
Effectively we would probably need to match to (parts of) the `compactFile` after we have fully constructed it. something along the lines of 
```python
detModel = os.path.basename(compactFile).split('.')[0]
if detModel in FCCeeMDI_DETECTOR_MODELS:
    # ....
```

Reply:
I have also thought about how to identify the different detector versions uniquely and `compactFile` paths are probably the best way. It might be a good idea to resolve the symlinks though